pkg/kubelet/prober/prober.go

```diff
@@ -110,19 +112,19 @@ func (pb *prober) probe(probeType probeType, pod *v1.Pod, status v1.PodStatus, c
 		if err != nil {
 			klog.V(1).Infof("%s probe for %q errored: %v", probeType, ctrName, err)
 			if hasRef {
-				pb.recorder.Eventf(ref, v1.EventTypeWarning, events.ContainerUnhealthy, "%s probe errored: %v", probeType, err)
+				pb.recorder.Eventf(ref, nil, v1.EventTypeWarning, containerUnhealthy, "%s probe errored: %v", "probe", err)
```

Comment:
Please don't do that - use the constant from pkg/kubelet/events instead of copying it to here.

Reply:
Same everywhere below.